<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#RegisterDialog {
	background-color: qlineargradient(spread:pad, x1:0.469, y1:0.188409, x2:0.598, y2:0.971682, stop:0 rgba(253, 234, 253, 255), stop:0.24581 rgba(200, 241, 254, 255), stop:0.692737 rgba(207, 255, 254, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="TitleWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>392</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="minimizeBtn">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#minimizeBtn {
	font-family: &quot;windowfont&quot;;
	border: none;
}

#minimizeBtn:hover {
	
	background-color: rgb(190, 190, 190);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeBtn">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>25</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#closeBtn {
	font-family: &quot;windowfont&quot;;
	border: none;
}

#closeBtn:hover {
	background-color: rgb(190, 0, 0);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="register_page">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>79</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,3,2,1">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget" native="true">
               <layout class="QVBoxLayout" name="verticalLayout">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <pointsize>20</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>欢迎注册MyMusic</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_3" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLineEdit" name="userLineE">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">/* 正常样式 */
#userLineE {
	border-radius: 5px;
	font-size: 18px;
	padding-left: 5px;
	padding-top: 5px;
	padding-bottom: 5px;
}

#userLineE[state = &quot;error&quot;] {
	border: 1px solid &quot;red&quot;;
}</string>
                  </property>
                  <property name="placeholderText">
                   <string>输入用户名</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="userTip">
                  <property name="styleSheet">
                   <string notr="true">#userTip {
	color: &quot;red&quot;;
}</string>
                  </property>
                  <property name="text">
                   <string>用户名不能为空</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_2" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>36</height>
                   </size>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLineEdit" name="pwdLineE">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>28</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>28</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">#pwdLineE {
	border-top-left-radius: 5px;
	border-bottom-left-radius: 5px;
	font-size: 18px;
	padding-left: 5px;
	padding-top: 5px;
	padding-bottom: 5px;
}

#pwdLineE[state = &quot;error&quot;] {
	border-top: 1px solid red;
    border-left: 1px solid red;
    border-bottom: 1px solid red;
}</string>
                     </property>
                     <property name="echoMode">
                      <enum>QLineEdit::Password</enum>
                     </property>
                     <property name="placeholderText">
                      <string>输入密码</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QPushButton" name="pwdVisibleBtn">
                     <property name="minimumSize">
                      <size>
                       <width>28</width>
                       <height>28</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>28</width>
                       <height>28</height>
                      </size>
                     </property>
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="focusPolicy">
                      <enum>Qt::NoFocus</enum>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">#pwdVisibleBtn {
	border: none;
	font-family: &quot;otherfont&quot;;
	font-size: 15px;
	background-color: &quot;white&quot;;
	border-top-right-radius: 5px;
	border-bottom-right-radius: 5px;
}

#pwdVisibleBtn[state = &quot;error&quot;] {
	border-top: 1px solid red;
    border-right: 1px solid red;
    border-bottom: 1px solid red;
}</string>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="checkable">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="pwdTip">
                  <property name="styleSheet">
                   <string notr="true">#pwdTip {
	color: &quot;red&quot;;
}</string>
                  </property>
                  <property name="text">
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="emailLineE">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>28</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">#emailLineE {
	border-radius: 5px;
	font-size: 18px;
	padding-left: 5px;
	padding-top: 5px;
	padding-bottom: 5px;
}

#emailLineE[state = &quot;error&quot;] {
	border: 1px solid red;
}</string>
                  </property>
                  <property name="placeholderText">
                   <string>输入邮箱</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="emailTip">
                  <property name="styleSheet">
                   <string notr="true">#emailTip {
	color: &quot;red&quot;;
}</string>
                  </property>
                  <property name="text">
                   <string>邮箱不能为空</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QWidget" name="widget_4" native="true">
                  <layout class="QHBoxLayout" name="horizontalLayout_2">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLineEdit" name="verifCodeLineE">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>28</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>28</height>
                      </size>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">#verifCodeLineE {
	border-top-left-radius: 5px;
	border-bottom-left-radius: 5px;
	font-size: 18px;
	padding-left: 5px;
	padding-top: 5px;
	padding-bottom: 5px;
}

#verifyCodeLineE[state = &quot;error&quot;] {
	border: 1px solid red;
}</string>
                     </property>
                     <property name="placeholderText">
                      <string>邮箱验证码</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="TimerButton" name="sendVerify">
                     <property name="minimumSize">
                      <size>
                       <width>70</width>
                       <height>28</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>70</width>
                       <height>28</height>
                      </size>
                     </property>
                     <property name="cursor">
                      <cursorShape>PointingHandCursor</cursorShape>
                     </property>
                     <property name="focusPolicy">
                      <enum>Qt::NoFocus</enum>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">#sendVerify {
	border: none;
	border-top-right-radius: 5px;
	border-bottom-right-radius: 5px;
	padding-right: 5px;
	color: rgb(136, 136, 204);
	background-color: &quot;white&quot;;
}

#sendVerify[state = &quot;error&quot;] {
	border: 1px solid red;
}</string>
                     </property>
                     <property name="text">
                      <string>发送验证码</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="verifyCodeTip">
                  <property name="styleSheet">
                   <string notr="true">#verifyCodeTip {
	color: &quot;red&quot;;
}</string>
                  </property>
                  <property name="text">
                   <string>邮箱验证码不能为空</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="registerBtn">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">#registerBtn {
	border: 2px;
	border-radius: 3px;
	font-size: 20px;
	color: white;
	background-color: rgb(76, 109, 255);
}</string>
               </property>
               <property name="text">
                <string>立即注册</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>79</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="finish_page">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>104</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget_8" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QWidget" name="widget_7" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="returnTip">
               <property name="styleSheet">
                <string notr="true">#returnTip {
	font-size: 15px;
	color: rgb(0, 227, 0);
}</string>
               </property>
               <property name="text">
                <string>注册成功，5秒后返回登录</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="styleSheet">
                <string notr="true">font-size: 15px;
color: rgb(0, 227, 0);</string>
               </property>
               <property name="text">
                <string>或点击返回登录</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_9" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3,1">
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>151</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="backBtn">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>-1</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">#backBtn {
	border: 1px;
	border-radius: 3px;
	font-size: 15px;
	color: white;
	background-color: rgb(76, 109, 255);
}</string>
               </property>
               <property name="text">
                <string>返回</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>54</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>104</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimerButton</class>
   <extends>QPushButton</extends>
   <header>timerbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
